### DISCUSSION TITLE
Errors on Nordigen import through FIDI

### DISCUSSION THREAD

**Discussion by Chacsam** [firefly-iii/firefly-iii#7334] [Firefly III]
When Importing from my bank through Nordigen, the base import works, but I get many skipped lines due to the same error

Line #2: Submission HTTP error: cURL error 52: Empty reply from server (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://mydomain/api/v1/transactions
Line #3: Submission HTTP error: cURL error 52: Empty reply from server (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://mydomain/api/v1/transactions
Line #5: Submission HTTP error: cURL error 52: Empty reply from server (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://mydomain/api/v1/transactions

Loading small volumes (7 days), so I guess that's not the issue.
If I run the same process 2 or 3 times, it will finally import everything as errors "randomly" appear on different transactions, but that's not an ideal way of working.

Is this a Nordigen or a FIDI/Firefly III issue?

**Comment by JC5** [comment]
What's the reverse proxy config like?

**Comment by Chacsam** [comment]
Hi,
I am using a Synology NAS reverse proxy, NGINX-based.
![image](https://user-images.githubusercontent.com/28670803/230754401-9628df80-d432-40e5-83a6-bedfb328b3bb.png)

with following advanced settings
![image](https://user-images.githubusercontent.com/28670803/230754342-4b2deb7e-98cb-4e7c-8458-fbc69fa02f47.png)

The importer is accessed locally only via IP:Port

**Reply by JC5** [reply]
Mmm sorry, I don't know much about Synology.

**Comment by Chacsam** [comment]
Thanks for checking

**Comment by Chacsam** [comment]
Hi all,
Update FYI only
As I wasn't happy with the whole performance, I just tried moving fireflyIII to an old laptop on ubuntu server.
No import error... So I guess it was about timeouts due to low performance on the Synology
